Tests/Packet++Test/Tests/SomeIpTpTests.cpp

```diff
@@ -0,0 +1,171 @@
+#include "../TestDefinition.h"
```

Comment:
Let's move these test cases to [`SomeIpTests.cpp`](https://github.com/seladb/PcapPlusPlus/blob/master/Tests/Packet%2B%2BTest/Tests/SomeIpTests.cpp)